src/components/ReadMore.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import { ReactNode } from 'react';
+
+import '../styles/ReadMore.scss';
+
+interface IProps {
+  children: ReactNode;
+}
+
+const ReadMore = ({ children }: IProps) => {
+  const text: any = children;
```

Comment:
any to React Node? It is typed above...

Reply:
there is an issue appears if not to specify "any" type for "text" variable 
<img width="849" alt="Screenshot 2021-04-13 at 10 51 09" src="https://user-images.githubusercontent.com/35579102/114518151-b9dfe000-9c47-11eb-9485-cb1f331d320a.png">

Reply:
<img width="742" alt="Screenshot 2021-04-13 at 16 51 23" src="https://user-images.githubusercontent.com/16467053/114563921-89646a00-9c78-11eb-9fcb-e69ec968dfa8.png">

Maybe change type to ReactText then, and later cast to string instead like const text = String(children)? if this works

Reply:
Any luck on this? :)

Reply:
no, an error appears during compilation
<img width="755" alt="Screenshot 2021-04-13 at 20 27 15" src="https://user-images.githubusercontent.com/35579102/114595380-fdae0600-9c96-11eb-9b17-dbac8edc03af.png">

Reply:
yes, your solution work, the reason why i got an error was to wrap into React.Fragment "return code" to return the single root element.
Thanks a lot!